akka-docs/src/main/paradox/stream/operators/Source-or-Flow/fold.md

```diff
@@ -19,6 +19,17 @@ completes, the current value is emitted downstream.
 
 Note that the `zero` value must be immutable.
```

Comment:
Maybe we should explain why it must be immutable?

, because otherwise the same mutable instance would be shared across different threads when running the stream more than once.